core/audits/accessibility/aria-tooltip-name.js

```diff
@@ -17,8 +17,8 @@ const UIStrings = {
   title: 'ARIA `tooltip` elements have accessible names',
   /** Title of an accessibility audit that evaluates if tooltip HTML elements do not have accessible names. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
   failureTitle: 'ARIA `tooltip` elements do not have accessible names.',
-  /** Description of a Lighthouse audit that tells the user *why* they should have accessible names for HTML elements. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'When an element doesn\'t have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn how to name `tooltip` elements](https://dequeuniversity.com/rules/axe/4.4/aria-tooltip-name).',
+  /** Description of a Lighthouse audit that tells the user *why* they should have accessible names for HTML 'tooltip' elements. This is displayed after a user expands the section to see more. No character length limits. 'Learn how...' becomes link text to additional documentation. */
+  description: 'When a tooltip element doesn\'t have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn how to name `tooltip` elements](https://dequeuniversity.com/rules/axe/4.4/aria-tooltip-name).',
```

Comment:
should we mark tooltip/meter with backticks?

Reply:
I took that from the axe docs (e.g. ["Aria tooltip elements must have discernible text..."](https://dequeuniversity.com/rules/axe/4.4/aria-tooltip-name#:~:text=Aria%20tooltip%20elements%20must%20have%20discernible%20text)), though to be fair, their docs language selector has only one other language and that particular string stays in english :)

I kind of justified it in my head as it's an aria role, but also a description of the type of element, so it makes some sense to include a localized name. `tooltip` (and `meter` for `aria-meter-name`) are backticked later in the string 🤷